torch/__init__.py

```diff
@@ -2275,11 +2275,12 @@ def compiled_with_cxx11_abi() -> builtins.bool:
 class _TorchCompileInductorWrapper:
     compiler_name = "inductor"
 
-    def __init__(self, mode, options, dynamic):
+    def __init__(self, mode, options, dynamic, name):
         from torch._inductor.compiler_bisector import CompilerBisector
 
         self.config: dict[str, _Any] = {}
         self.dynamic = dynamic
+        self.name = name
```

Comment:
I expect that changing the torch.compile() API will require more people to look at this than just me haha, cc @ezyang